docusaurus/docs/key-concepts/ui-extensions.md

```diff
@@ -48,7 +48,7 @@ As a content producer, you can either **expose** your content to all consumers o
 
 Use **expose APIs** when you want to make something available **broadly**, without knowing in advance who will consume it.
 
-- **Producer perspective:** App plugins or Grafana publish (expose) a component, function, or link so that **any app plugin** can discover and consume it.
+- **Producer perspective:** App plugins or Grafana expose a component, function, or link so that **any app plugin** can discover and consume it.
```

Comment:
are we sure we should mention `Grafana expose a component`? I know we do but should we add some more context to this?